main.py

```diff
@@ -1595,21 +1595,48 @@ def get_smmry(message):
 
 async def whoami(client, message):
     """
-    Handles !whoami
+    Handles !me
 
     :param client: The Client
     :param message: The message
     :return: None
     """
     author = str(message.author.display_name)
     if author in users and users[author] != {}:
-        await client.send_message(message.channel,
-                                  "Well, I don't know you that well, but "
-                                  "from what I've been hearing on the "
-                                  "streets...")
+        message_output = "Well, I don't know you that well, but " \
+                         "from what I've been hearing on the " \
+                         "streets...\n"
+
         for k, v in users[author].items():
-            await client.send_message(message.channel,
-                                      "Your {} is {}.".format(k, v))
+            if k == "duel_record":
+                if v[0] < 10:
+                    output = "You're a pretty green dueler"
+                elif v[0] < 100:
+                    output = "You're a seasoned dueler"
+                else:
+                    output = "You're a master dueler"
+
+                output += ", and your record is **{}** wins, **{}** losses," \
+                          " and **{}** ties.".format(v[0], v[1], v[2])
+            elif k == "a_item":
+                if v == None:
+                    output = "You don't have a dueling item equipped."
+                else:
+                    output = "You have **{}** equipped.".format(DuelItem(0, v).name)
+            elif k == "inventory":
+                # TODO: display_inventory
+                if v == {}:
+                    output = "You don't have an inventory for dueling items."
+                else:
+                    output = "Your inventory of dueling items:"
+                    for item, count in v.items():
```

Comment:
Count here is the number of times the item has been used. Was this intended? What you have is a print for:
`Times Used: Name of Item`